src/blocks/product-card.css

```diff
@@ -0,0 +1,94 @@
+.product-card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  box-sizing: border-box;
```

Comment:
move it to the * selector

Reply:
when  I move, then the block of starts doesn't work correctly anymore.

Reply:
![1](https://user-images.githubusercontent.com/59209337/209525819-82dd2f64-7f14-4c4d-9022-cb052cbe309a.jpg)
![2](https://user-images.githubusercontent.com/59209337/209525820-214506df-a874-449b-b544-7398f489b38f.jpg)
I try it in dev tools - result the same